docs-kits/kits/ESS-Kit/page_adoption-view.md

```diff
@@ -163,13 +171,179 @@ The data model follows the Catena-X Standard CX-0113 (<https://catena-x.net/de/s
+
+In the Catena-X use case sustainability / environmental and social standards, principles are established with the aim of establishing transparent and trustworthy practices within the Catena-X network.
+These principles are based on national and international legal regulations and are aligned with internationally recognised standards such as the International Bill of Human Rights, the UN Guiding Principles on Business and Human Rights, the OECD Guidelines for Multinational Enterprises, as well as the ILO core labour standards and the principles of the UN Global Compact.
+Accordingly, this document defines the minimum requirements and clear expectations for the corporate due diligence obligations of the members. We expect you, as our member, to ensure that these requirements and expectations are also passed on appropriately to your downstream business partners. Acting responsibly and lawfully is therefore an integral part of participating in the Catena-X network.
+The Catena-X network includes companies and organizations that design, manufacture, market or provide goods and services for the automotive industry. The Code of Conduct must be adopted by every company participating in the Catena-X Use Case Sustainability / Environmental and Social Standards in a binding manner or guaranteed by an appropriate standard of its own. The recognition of the Catena-X Code of Conduct is the basis of a cooperative business relationship with our members.  Given the complexity and dynamics of the n-tier network, we rely on joint activities with all our members to identify risks to the environment and society, create more transparency and achieve more effectiveness.
+The Catena-X Use Case Sustainability / Environmental and Social Standards participants should consider the Code as a comprehensive supply chain initiative. A prerequisite for membership is conducting business activities in full compliance with the law.
+The Code also encourages participants to go beyond legal compliance and rely on internationally recognized standards to promote social and environmental responsibility and business ethics, for example by increasing their own sphere of influence through complementary measures or collaborations, and by encouraging collaboration with non-governmental organizations.
+The Catena-X Use Case Sustainability / Environmental and Social Standards is committed to regularly engaging with stakeholders in the ongoing development and implementation of the Code of Conduct.
+The Code consists of three sections. Section I contains the requirements for Catena-X members. Section II describes the proposed implementation of the Catena-X Code of Conduct. Section III refers to the elements of an acceptable system that complies with this Code of Conduct.
+
+#### I Requirements for Catena-X members
+
+The following requirements for Catena-X members are derived, among other things, from the internationally recognized guiding principles.
+
+#### 1 Responsible Business practices

```

Comment:
Just a recommendation. I would remove the numbering. A hierarchy is given anyway by the headings ( ## ### ...). The numbers only causes more effort if one is removed.

Reply:
The part with the "Code of Conduct" is like a separate document included in the kit (as aligned in our use case), I would like to keep the numbering if somenow possible.